### PR TITLE
updated wms list

### PR DESCRIPTION
**Pull request by davidoesch**
Based on 
https://docs.google.com/spreadsheets/d/1BHAzi9__ZCgPYLpDmezyX3oy3AuwiPZUMnKrOTbjKv8/edit#gid=0&fvid=1913726399
fixed and removed a lot of wms

[Test link](https://mf-geoadmin3.int.bgdi.ch/[BRANCH_NAME]/index.html)
This link will work only if the build succeed

### CODE REVIEW

File: src/js/ImportController.js [hunk 1]
```diff
@@ -53,15 +50,13 @@ goog.require('ga_wmts_service');
         'http://wms.geo.gr.ch/wms/amtlichevermessung_stand',
         'http://wms.geo.gr.ch/wms/wildruhezonen',
         'http://wms.geo.gr.ch/wms/wildschutzgebiete',
-        'http://wms.geo.gr.ch/wms/zonenplan',
         'http://www.sogis1.so.ch/wms/avwms',
         'http://www.sogis1.so.ch/wms/grundbuchplan',
         'http://www.sogis1.so.ch/wms/wms_lidar',
         'http://wms.vd.ch/public/services/wmsVD/Mapserver/Wmsserver',
-        'https://wms.geo.bs.ch/wmsBS',
+        'http://wms.geo.bs.ch/wmsBS?',
```
Comment (oterral) [inline src/js/ImportController.js]: ? is not necessary

File: src/js/ImportController.js [hunk 2]
```diff
@@ -164,7 +155,8 @@ goog.require('ga_wmts_service');
         // non-SwissProjected test urls
         'http://wms.ga.admin.ch/1GE',
         'http://wms.ga.admin.ch/LG_DE_Geologie_und_Tektonik/wms',
-        'http://discomap.eea.europa.eu/arcgis/services/Land/CLC2000_Dyna_WM/MapServer/WMSServer',
+        'http://bio.discomap.eea.europa.eu/arcgis/services/Ecosystem/Ecosystems/MapServer/WMSServer?request=GetCapabilities',
```
Comment (oterral) [inline src/js/ImportController.js]: no need to write  "?request=GetCapabilities"

File: src/js/ImportController.js [hunk 3]
```diff
@@ -164,7 +155,8 @@ goog.require('ga_wmts_service');
...
+        'http://copernicus.discomap.eea.europa.eu/arcgis/services/Corine/CLC2012_WM/MapServer/WMSServer?request=GetCapabilities'
```
Comment (oterral) [inline src/js/ImportController.js]: Here too

File: src/js/ImportController.js (hunk 1 above)
Comment (davidoesch) [reply src/js/ImportController.js]: it _IS_ necessary ..othrwise it wont load in the pulldown....see the curretn implementation

File: src/js/ImportController.js (hunk 2 above)
Comment (davidoesch) [reply src/js/ImportController.js]: agree

File: src/js/ImportController.js (hunk 3 above)
Comment (davidoesch) [reply src/js/ImportController.js]: agree

File: src/js/ImportController.js (hunk 1 above)
Comment (oterral) [reply src/js/ImportController.js]: I've just tested on prod it works without. I think the WMS is just very  unstable.

Comment (davidoesch) [reply src/js/ImportController.js]: according to OWSLib this one works always with ? ....

**Pull request merged**